apps/docs/content/docs/frameworks/nextjs.mdx

```diff
@@ -30,14 +30,19 @@ directly in your React Server Components (RSC).
 If you are starting a new project, you can use the NextUI CLI to create a new project with NextUI pre-configured:
 
 ```bash
-npm install -g nextui-cli
+nextui init -t app
```

Comment:
suggest to link `NextUI CLI` in L30 to https://nextui.org/docs/guide/cli, so that people know how to install `nextui` before executing this line.

Reply:
Done in: [bae581b](https://github.com/nextui-org/nextui/pull/4410/commits/bae581b032bfba5c71db0d6da112940a8da5a837)